README.md

```diff
@@ -4,6 +4,8 @@ Low-level, unsafe Rust bindings for the [`capstone`][capstone] disassembly libra
 
 [capstone]: https://github.com/aquynh/capstone
 
+**[API Documentation](https://docs.rs/capstone-sys/)**
```

Comment:
Is this format acceptable?

Reply:
Sorry for delay. 

I'm not sure what you mean ? The doc reference ? If so, yes

If it's changes to the documented functions I don't see any changes in this PR